src/rcc.rs

```diff
@@ -325,13 +325,13 @@ impl Clocks {
         self.pclk2
     }
 
-    pub(crate) fn ppre1(&self) -> u8 {
+    pub fn ppre1(&self) -> u8 {
```

Comment:
Why were these made public? As far as  I can tell, the are not used anywhere in this PR